lib/premonition/processor.rb

```diff
@@ -52,7 +52,9 @@ def adder(content)
       def load_references(content)
         refs = ["\n"]
         content.each_line do |l|
-          refs << l if l.strip!.match(/^\[.*\]:.*\".*\"$/i)
+          unless l.nil? || l == 0 || (l.is_a? String) 
+          refs << l if l.to_s.strip!.match(/^\[.*\]:.*\".*\"$/i)
```

Comment:
Nitpick: this line needs to be indented

Reply:
@RobbiNespu Could you provide what @MaximeKjaer asked for? I would also like to be able to reproduce this in some test cases.

Reply:
@MaximeKjaer @lazee sorry, did't notice there is review msg on this. I re-indent that line as require

Reply:
@RobbiNespu Great, but could you also provide an example of what caused this, so that we can add a test for it?